src/ui/story.js

```diff
@@ -47,24 +47,26 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             // ],
             // translationOf: null,
         },
-        modelRelay: [{
-            target: "contentString",
-            singleTransform: {
-                type: "sjrk.storyTelling.transforms.arrayToString",
-                input: "{that}.model.content",
-                separator: ". ",
-                stringOnly: true,
-                path: "contentString"
+        modelRelay: {
+            contentToContentString: {
+                target: "contentString",
+                singleTransform: {
+                    type: "sjrk.storyTelling.transforms.arrayToString",
+                    input: "{that}.model.content",
```

Comment:
Instead of specifying `input`, is it possible to just add this as the modelRelay `source`?